From fba8b564fde70384b98d09009286ffd4e63eb175 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 26 Aug 2020 08:08:05 +0800
Subject: [Patch 08/10] Support for INF [Sources.X64] test cases.

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/FakeLogo/JpegLogoDxe.inf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/FakeLogo/JpegLogoDxe.inf b/TestPkg/FakeLogo/JpegLogoDxe.inf
index 3c536ef0fc..a3af409553 100644
--- a/TestPkg/FakeLogo/JpegLogoDxe.inf
+++ b/TestPkg/FakeLogo/JpegLogoDxe.inf
@@ -25,11 +25,11 @@
 # The following information is for reference only and not required by the build tools.
 #
 #  VALID_ARCHITECTURES           = IA32 X64
 #
 
-[Sources]
+[Sources.X64]
   Logo.jpg
   Logo.c
   JpegLogo.idf
 
 [Packages]
-- 
2.29.1.windows.1

